modules/ROOT/pages/db-util.adoc

```diff
@@ -273,13 +144,11 @@ To do this, you use the `validate-that` operation. Set the `results` property to
 query="SELECT * FROM jobtitlelookup WHERE JOBTITLE='Developer';" returns="&quot;JOBTITLE&quot;,&quot;EECSALARYORHOURLY&quot;,&quot;JOBTITLEID&quot;\n&quot;Developer&quot;,&quot;10&quot;,&quot;DEV&quot;" />
 ----
 
-=== Executing SQL instructions and SQL queries
+=== Execute
 
-The MUnit DB Server allows you to execute instructions on the in-memory databases, so
-you can add or remove registries before a test, and also check if your data was
-stored correctly.
+The MUnit DB Server allows you to execute instructions on the in-memory databases, so you can add or remove registries before a test, and also check if your data was stored correctly.
 
-Use a DB Server connector with the `execute` operation to insert the values new values to the table (`Culinary Team Member,10,HIR`), then use another DB Server connector with an `execute query` operation to retrieve all values from the table (including the newly added ones) and check the payload using a logger component to store the payload.
+Use a DB Server connector with the `execute` operation to insert the values new values to the table (`Culinary Team Member,10,HIR`), then use another DB Server connector with an `execute query` operation to retrieve all values from the table (including the newly added ones) and check the payload using a logger component to store the payload:
```

Comment:
DB Server Connector? Or maybe we should be referring to the full name, MUnit DB Server, so that readers don't confuse it, as I apparently have, with some generic database server.